README.md

```diff
@@ -4,7 +4,13 @@ Please see website at https://xilinx.github.io/fpga24_routing_contest.
 
 Please report all issues using the GitHub [Issues](https://github.com/Xilinx/fpga24_routing_contest/issues) feature, and ask questions under [Discussions](https://github.com/Xilinx/fpga24_routing_contest/discussions).
 
-All content in this repository except for third-party software (Gradle) is licensed under:
+---
+All content in this repository except for third-party software ([Gradle](https://gradle.org)) is licensed under:
 ```
 SPDX-License-Identifier: MIT
 ```
+---
+
+Utilities provided:
+* [`net_printer`](https://github.com/Xilinx/fpga24_routing_contest/tree/dcp_to_fpgaif/net_printer) -- inspect the routing of nets in a Physical Netlist.
```

Comment:
Is there a reason that this links to the `dcp_to_fpgaif` branch instead of `master`?

Reply:
Good catch! Will fix it now.